### PR TITLE
Fingerprinting assets

### PR DESCRIPTION
**Pull request by peter-mouland**
these assets will dow be available on the `http://toga.server.url/v1/` end point

to know what the latest version of assets will be, you will need to hit the `http://toga.server.url/asset-bundles` which returns a JSON object with the file names.

take a look at the examples app to see this in action

### CODE REVIEW

**Comment by peter-mouland** [comment]
- [ ] add fingerprint to html response

File: src/script/deployBundles.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        // See: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html#constructor-property
+  }
+});
+
+const params = {
+  localDir: 'dist/components',
+  s3Params: {
+    ACL:'public-read',
+    Bucket: aws.bucket,
+    Prefix: 'toga-assets/'
```
Comment (chrisface) [inline src/script/deployBundles.js]: Don't forget `CacheControl` :)

File: docker-compose-testing.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3'
```
Comment (teofilomonteiro) [inline docker-compose-testing.yml]: Why are you using this '3' ? Shouldn't this be '2' ?

File: docker-compose-dev.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3'
```
Comment (teofilomonteiro) [inline docker-compose-dev.yml]: Why are you using this '3' ? Shouldn't this be '2' ?

File: docker-compose.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3'
```
Comment (teofilomonteiro) [inline docker-compose.yml]: Same as above

**Pull request merged**